ISPN-12921 Upgrade 12.1.0.Final (#117)
infinispan/infinispan-simple-tutorials: README.md

# Infinispan Simple Tutorials  

[![Build Status](https://travis-ci.org/infinispan/infinispan-simple-tutorials.svg?branch=master)](https://travis-ci.org/infinispan/infinispan-simple-tutorials)

This is a collection of simple tutorials that explain how to use certain
features of Infinispan in the most straightforward way possible.

## Infinispan Client/Server tutorials

Some tutorials require the Infinispan Server running locally.

To run the Server with a container image, check the "Get started" guide 
in the Infinispan Website.
[Getting started with Infinispan](http://infinispan.org)

> You can always [download](download link) the latest server distribution and run

Reviewer: missing download link

from the downloaded files:

```bash

> ./bin/cli.sh user create admin -p "pass"

Reviewer 2: ```suggestion
./bin/cli.sh user create admin -p "password"
```

./bin/server.sh
```
*IMPORTANT:* 
**Authentication and authorization are implicit since Infinispan 12.1.** 

> Creating a `admin` user, will grant the user with the `admin` role.

Reviewer 2: ```suggestion
Creating a user named `admin` gives you administrative access to Infinispan Server.
```

> Check more about implicit authorization in [Authorization Guide](documentation)

Reviewer: and here :)

> Check more about implicit authorization in [Authorization Guide](documentation)

Reviewer 2: ```suggestion
You can find more details about users and permissions in [Creating and Modifying Users](https://infinispan.org/docs/stable/titles/server/server.html#creating-users_quickstart)
```
